utoipa-swagger-ui/src/lib.rs

```diff
@@ -758,6 +766,24 @@ impl<'a> Config<'a> {
         self
     }
 
+    /// Set 'layout' to 'BaseLayout' to only use the base swagger layout without a search header.
+    ///
+    /// Default value is 'StandaloneLayout'.
+    ///
+    /// # Examples
+    ///
+    /// Allow operation id to be shown.
+    /// ```rust
+    /// # use utoipa_swagger_ui::Config;
+    /// let config = Config::new(["/api-doc/openapi.json"])
+    ///     .use_base_layout();
+    /// ```
+    pub fn use_base_layout(mut self) -> Self {
+        self.layout = SWAGGER_BASE_LAYOUT.to_string();
```

Comment:
And just a reference here :)
```suggestion
        self.layout = SWAGGER_BASE_LAYOUT;
```